### ISSUE TITLE
Not possible to answer to multiple persons

### ISSUE THREAD

**Issue by Astaoth** [poetaster/harbour-tooter#39]
Hi,

When I answer to a toot, the previous toot writter is automatically mentionned at the beginning, unless it's me. This is the normal behaviour and it's good.

However, when we are multiple persons in the discussion, with multiple @<name> at the beginning of the previous toot (or the end, it depends of how others use mastodon), they aren't automatically added. I have to writte the missing @<name> (which are everyones outside of the previous toot writter) myself.
It's especially annoying because I'm often in discussions with multiple people, and they are some of them which have changed of servers (they appear under multiple servers when looking for them) and others which have similar names than other people.

Would it be possible, when answering to a toot which mentions multiple people, to prefill the answer by mentionning all of them ?

**Comment by poetaster** [comment]
That should be possible.

**Comment by Astaoth** [comment]
Hi, I've made few tests, and this doesn't work for me. The only user included in the answer, is the owner of the toot to which I'm answering, and not the others in the same thread.

I've TooterB 1.1.9-1.15.1.jolla installed from Chum, with SFOS 4.5.0.19 aarch64, what could I've missed for getting this working as it should ?

**Comment by poetaster** [comment]
> Hi, I've made few tests, and this doesn't work for me. The only user included in the answer, is the owner of the toot to which I'm answering, and not the others in the same thread.
> 
> I've TooterB 1.1.9-1.15.1.jolla installed from Chum, with SFOS 4.5.0.19 aarch64, what could I've missed for getting this working as it should ?

No, I think it just is limited in tooter at the moment. You're not doing anything wrong :) 

I meant it should be possible to implement :) Now, the how is a bit tricky. Answering to ALL people in the thread could be wrong for a number of reasons. However, for a toot with multiple @s all those should show up in the reply to the toot. 

I'm working on it!

**Comment by Astaoth** [comment]
Yeah, by "The only user included in the answer, is the owner of the toot to which I'm answering, and not the others in the same thread." I meant other people @ in the previous toot, not everyone which have been mentioned at some point in the thread.

Thanks for taking a look at this !

**Comment by poetaster** [comment]
> Yeah, by "The only user included in the answer, is the owner of the toot to which I'm answering, and not the others in the same thread." I meant other people @ in the previous toot, not everyone which have been mentioned at some point in the thread.

Ok, that's the thing I wanted to fix :)

> Thanks for taking a look at this !

Oh, it was bothering me too. I just fixed the display of images in a conversation and sometimes, there's an something missing between! I've almost got it sorted :)

**Comment by poetaster** [comment]
> Thanks for taking the time to look into this, but I'm not sure what you mean by a quoted response. If you mean puting the whole previous message into the one I'm writting as a quote, I don't always want to put the previous message in mine because of the character limits and for the readability. I'm on a server were the char limit is much higher than the one allowed by Tooter, I wouldn't be even able to send my message.

Ok. Cool. So it's just making sure that reply catches the @ persons. The question is should all '@' in a post count, or only the first ones? It's not so clear in the api/spec but I would guess just consider the first ones?

**Comment by poetaster** [comment]
implemented in https://github.com/poetaster/harbour-tooter/tree/laril
